Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM python:2.7
+FROM python:3.6
 ENV PYTHONUNBUFFERED 1
+
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+&& apt-get update && apt-get install -y gettext postgresql-client nodejs \
+&& wget https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -P /usr/bin \
+&& chmod +x /usr/bin/wait-for-it.sh
+
 RUN mkdir -p /usr/src/app
-RUN apt-get update && apt-get install -y gettext postgresql-client
-RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -P /usr/bin
-RUN chmod +x /usr/bin/wait-for-it.sh
+
 WORKDIR /usr/src/app
-ADD requirements.txt /usr/src/app/
+ADD requirements.txt package.json /usr/src/app/ 
```

Comment:
COPY instead of ADD should be used. ADD should only be used when an archive should be extracted into docker container. Below are some more ADDs